src/app/content/components/Page/contentLinkHandler.ts

```diff
@@ -138,7 +138,7 @@ export const contentLinkHandler = (anchor: HTMLAnchorElement, getProps: () => Co
 
     e.preventDefault();
 
-    if (isHtmlElementWithHighlight(e.target)) {
+    if (isHtmlElementWithHighlight(e.target) && !e.target.closest('a')) {
```

Comment:
we're in the link click handler, i don't think there'll ever _not_ be a closest anchor. this change makes it impossible to select a highlight that contains only an anchor.

i think it would work to change the conditional below `e.target.getAttribute('data-highlight-id') !==  focusedHighlight` to not rely on the `focusedHightlight` state and instead use the css, like `!e.target.matches('.focus')`, that way it'll match either a focused user highlight or a focused search highlight

Reply:
That does make the highlight clickable, but breaks [a test](https://github.com/openstax/rex-web/blob/main/src/app/content/components/Page/contentLinkHandler.spec.ts#L293).

Reply:
because the test is written to rely on the redux state [here](https://github.com/openstax/rex-web/blob/main/src/app/content/components/Page/contentLinkHandler.spec.ts#L288) instead of the class, it doesn't seem like this test is integrating the entire highlighter, so you may have to add the `focus` class manually to the element that it [creates](https://github.com/openstax/rex-web/blob/main/src/app/content/components/Page/contentLinkHandler.spec.ts#L256)

Reply:
That does make the test happy.